controller/linodemachine_controller.go

```diff
@@ -70,6 +70,11 @@ var requeueInstanceStatuses = map[linodego.InstanceStatus]bool{
 	linodego.InstanceResizing:     true,
 }
 
+type nodeIP struct {
+	ip     string
+	ipType clusterv1.MachineAddressType
+}
```

Comment:
nit: Replace this with [`clusterv1.MachineAddress`](https://pkg.go.dev/sigs.k8s.io/cluster-api/api/v1beta1#MachineAddress)

Reply:
Updated